server/constants.py

```diff
@@ -0,0 +1,24 @@
+BUSINESS_PRODUCTS = [
+    "bigTexMonthly",
+    "bigTexYearly",
+    "loneStarMonthly",
+    "loneStarYearly",
+    "hatsOffMonthly",
+    "hatsOffYearly",
+]
+
+CIRCLE_PRODUCTS = [
+    "foundersMonthly",
+    "foundersYearly",
+    "chairmanMonthly",
+    "chairmanYearly",
+    "leadershipMonthly",
+    "leadershipYearly",
+    "editorMonthly",
+    "editorYearly", 
+]
+
+BLAST_PRODUCTS = [
+    "blast",
+    "blastTaxExempt",
+]
```

Comment:
**thought:** I think you speak to this a little in the PR notes, but does this sorta create two sources of truth for these names? Or make it so that we're maintaining them in two places I mean. Not sure of a way around that but it just feels a little funny.

I'm referring to these other instances of them:
https://github.com/texastribune/donations/blob/bcb441aef4b165e80a1761ce3894d9c4c66eeb62/server/config.py#L66-L84 right?

Reply:
mmm, good point... I made some updates to the STRIPE_PRODUCTS dict to simplify this a bit... let me know what you think

Reply:
ah yeah that makes sense to me! I guess that means you'll have to refactor how we're calling that in a few other places too right? I'm seeing [this error](https://the-texas-tribune.sentry.io/issues/4554433156/?alert_rule_id=1608736&alert_type=issue&notification_uuid=ed6f399a-715a-4b80-8bb9-cdefe1b793e0&project=1304832) when I submit using the form.

Reply:
Alright alright alright, I ended up making this much simpler and moving the donation type and the donor_amount (basically the base amount sans fees) in stripe. I think it makes sense to make that the source of truth for that data, but let me know what you think @ashley-hebler.

Reply:
oh ok yeah I think I follow and that sounds even better!